File: Web/RMA-Member-Portal/memberportal-app/src/app/shared/models/brokerage-product-option.ts
import { ProductOption } from "./product-option";

export class BrokerageProductOption {
  id: number;
  brokerageId: number;
  productOptionId: number;
  startDate: Date;
  endDate: Date;
  productOption: ProductOption;

  constructor(id: number, brokerageId: number, productOptionId: number, startDate: Date, endDate: Date, productOption: ProductOption) {
    this.id = id;
    this.brokerageId = brokerageId;
    this.productOptionId = productOptionId;
    this.startDate = startDate;
    this.endDate = endDate;
    this.productOption = productOption;
  }
}

File: Service/ClientCare/RMA.Service.ClientCare.Contracts/Entities/Broker/BrokerageProductOption.cs
using RMA.Service.ClientCare.Contracts.Entities.Product;

namespace RMA.Service.ClientCare.Contracts.Entities.Broker
{
    public class BrokerageProductOption
    {
        public int Id { get; set; }
        public int BrokerageId { get; set; }
        public int ProductOptionId { get; set; }
        public System.DateTime StartDate { get; set; }
        public System.DateTime? EndDate { get; set; }

        public ProductOption ProductOption { get; set; }
    }
}
